create-plugins-list-file.sh

```
@@ -36,6 +36,60 @@ get_plugin_name() {
+
+    # Ensure the temporary file is removed if the function exits unexpectedly
+    trap 'rm -f "$temp_file"' EXIT
+
+    if ! xsltproc remove-namespaces.xsl "$pom_file" > "$temp_file" 2>/dev/null; then
+        rm -f "$temp_file"
+        echo "Error: Failed to transform XML file" >&2
+        return 1
+    fi
+
+    local jenkins_core_version=""
+    for xpath in "${pom_xml_jenkins_core_version_xpath[@]}"; do
```

Shellcheck (0.10.0)

[warning] pom_xml_jenkins_core_version_xpath is referenced but not assigned.

(SC2154)